alembic

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+import sys
+from alembic.config import Config, CommandLine
+from webapp.database import get_database_url
+
+cmdline = CommandLine()
+options = cmdline.parser.parse_args(sys.argv[1:])
+
+config = Config()
+config.set_main_option("script_location", "migrations")
+config.set_main_option("sqlalchemy.url", get_database_url())
```

Comment:
It seems a little odd to need to set this URL here as well as in `webapp.database`. The `get_database_url` function seems like it should be redundant - we should only need to run `os.getenv("DB_URL", "sqlite:///db.sqlite")` once.

Reply:
alembic creates it's own db session, so I need to pass it the DB_URL. I can make it a `DB_URL` constant in `webapp.database` though.

Reply:
Yeah this is a weird architectural choice of alembic. You would think it could just use the DB engine/session that the actual app uses.